src/riakc_ts_get_operator.erl

```diff
@@ -46,6 +46,8 @@ deserialize({error, {Code, Message}}) when is_integer(Code), is_atom(Message) ->
     {error, {Code, iolist_to_binary(atom_to_list(Message))}};
 deserialize({error, Message}) ->
     {error, Message};
+deserialize(tsgetresp) ->
+    {ok, {[], []}};
```

Comment:
@hmmr Looks like we had similar solutions to this problem: https://github.com/basho/riak-erlang-client/pull/276  Adding the second head I added (with empty rows) would make it consistent with `riakc_ts_query_operator:deserialize`.